src/scenario.cpp

```diff
@@ -438,45 +441,46 @@ bool scenario::custom_initial_date() const
     return _custom_initial_date;
 }
 
-bool scenario::random_initial_hour() const
+bool scenario::random_hour() const
 {
-    return _random_initial_hour;
+    return _initial_hour == -1;
 }
 
-bool scenario::random_initial_day() const
+bool scenario::random_day() const
 {
-    return _random_initial_day;
+    return _initial_day == -1;
 }
 
-bool scenario::random_initial_season( )const
+bool scenario::random_year() const
```

Comment:
Maybe call them `is_random_...`? This will make it clearer, imo.

Reply:
Sure.